File: .NET/Deviations/GetDeviation.cs
using System.Net.Http.Headers;
using System.Text;
using System.Text.Json;

var boardsOnFireApiUrl = "YOUR URL"; // exempel: https://my-company.boardsonfireapp.com/api/v5/
var apiKey = "YOUR API KEY";
var deviationId = "YOUR DEVIATION ID";

using (var client = new HttpClient())
{
    client.BaseAddress = new Uri(boardsOnFireApiUrl);
    client.DefaultRequestHeaders.Accept.Clear();
    client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
    client.DefaultRequestHeaders.Add("x-api-key", apiKey);

    try
    {
        var response = await client.GetAsync($"deviations/{deviationId}");
        response.EnsureSuccessStatusCode();

        var data = await response.Content.ReadAsStringAsync();
        Console.WriteLine("Deviation:");
        Console.WriteLine(data);
    }
    catch (HttpRequestException e)
    {
        Console.WriteLine($"Request error: {e.Message}");
    }
}

File: Python/Deviations/GetDeviation.py
import requests

boards_on_fire_api_url = "YOUR URL"  # example: https://my-company.boardsonfireapp.com/api/v5/
api_key = "YOUR API KEY"
deviation_id = "YOUR DEVIATION ID"

headers = {
    "x-api-key": api_key
}

try:
    response = requests.get(f"{boards_on_fire_api_url}/deviations/{deviation_id}", headers=headers)
    response.raise_for_status()
    data = response.json()
    print("Deviation:")
    print(data)
except requests.exceptions.RequestException as e:
    print(f"Request error: {e}")
